draft-ietf-taps-arch.md

```diff
@@ -110,7 +110,7 @@ they appear in all capitals, as shown here.
 
 ## Glossary of Key Terms
 
-This subsection provides a glossary of key terms related to the Transport Services architecture.
+This subsection provides a glossary of key terms related to the Transport Services architecture. It provides a simple description of key terms that are later defined in this document.
```

Comment:
Would "summary" be better than "simple description"?

Reply:
Would removing "simple" be a useful step in this direction?

Reply:
Yes, "simple" was the word that triggered my response, as I did not find that it read well. I am fine with just removing "simple". If you want another word, perhaps we could use "short" instead of "simple", but just removing "simple" is the easiest.

Reply:
I have tried /simple/short/ at your suggestion.